src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -48,9 +48,24 @@ public CommandResult execute() {
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
+
+        final ArrayList<String> lower_keywords = new ArrayList<>();
+        //get a lowercase keywords list
+        for(String newSplitWords : keywords)
+        {
+            //System.out.println(newSplitWords);
+            lower_keywords.add(newSplitWords.toLowerCase());
+        }
+
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final ArrayList<String> lower_wordsInName = new ArrayList<>();
+            for(String inWord : wordsInName)
```

Comment:
maybe a better name for inWord would be the singular of wordsInName which is wordInName

Reply:
thanks!